braa: add page (#9041)
tldr-pages/tldr: pages/linux/braa.md

# braa

> > Ultra fast Mass SNMP scanner allowing multiple hosts simultaneously. 

Reviewer: ```suggestion
> Ultra-fast mass SNMP scanner allowing multiple hosts simultaneously. 
```

> > More information: <https://github.com/mteg/braa>. 

Reviewer: ```suggestion
> More information: <https://github.com/mteg/braa>.
```

> - Walk the SNMP tree of host with public string querying all OIDs under .1.3.6:

Reviewer 2: ```suggestion
- Walk the SNMP tree of host with public string querying all OIDs under `.1.3.6`:
```

> `braa public@{ip}:.1.3.6.*`

Reviewer 2: Did you want to use the placeholder syntax to highlight `ip`? Or does the command require the `{ }`?
Also, we use the placeholder syntax for all variable arguments, even if it's already a specific example.
```suggestion
`braa public@{{ip}}:{{.1.3.6.*}}`
```

> - Query the whole subnet {ip_range} for system.sysLocation.0

Reviewer: ```suggestion
- Query the whole subnet `{ip_range}` for `system.sysLocation.0`:
```
I am not sure about this command, I will ask other's opinions.

> - Query the whole subnet `{ip_range}` for `system.sysLocation.0`:

Reviewer 2: ```suggestion
- Query the whole subnet `ip_range` for `system.sysLocation.0`:
```

> `braa public@{ip_range}:.1.3.6.1.2.1.1.6.0`

Reviewer 2: ```suggestion
`braa public@{{ip_range}}:{{.1.3.6.1.2.1.1.6.0}}`
```

> - Attempt to set the value of system.sysLocation.0 to "My Workgroup"

Reviewer: ```suggestion
- Attempt to set the value of `system.sysLocation.0` to "My Workgroup":
```

> - Attempt to set the value of `system.sysLocation.0` to "My Workgroup":

Reviewer 3: ```suggestion
- Attempt to set the value of `system.sysLocation.0` to a specific workgroup:
```

> `braa private@{ip}:.1.3.6.1.2.1.1.6.0=sMy Workgroup`

Reviewer 2: Don't you need to put `My Workgroup` in `'...'`, since it contains a space?
```suggestion
`braa private@{{ip}}:{{.1.3.6.1.2.1.1.6.0}}=s{{My Workgroup}}`
```

> `braa private@{{ip}}:{{.1.3.6.1.2.1.1.6.0}}=s{{'My Workgroup'}}`

Reviewer 3: We prefer quotes outside token syntax
```suggestion
`braa private@{{ip}}:{{.1.3.6.1.2.1.1.6.0}}=s'{{workgroup}}'`
```
